crates/router/src/core/refunds.rs

```diff
@@ -532,6 +530,7 @@ pub async fn validate_and_create_refund(
     })?;
 
     let refund = match validator::validate_uniqueness_of_refund_id_against_merchant_id(
+        // this is not required ? as (refund_id, merchant_id ) unique btree, just insert if insert fails not valid
```

Comment:
Create an issue for it and remove this comment

Reply:
Done #1521